Horn.fs

```diff
@@ -117,6 +116,7 @@ module Pretty =
         | AMul [ x; y ] -> printBop "*" x y
         | AMul(x :: y :: xs) -> printInt (AMul((AMul [ x; y ]) :: xs))
         | ADiv(x, y) -> printBop "/" x y
+        | AMod(x, y) -> failwith "unexpected modulo"
```

Comment:
I don't think there will be—HSF doesn't do nonlinear arithmetic as far as I know